### ISSUE TITLE
Sorry, Not Authenticated

### ISSUE THREAD

**Issue by IvRRimum** [yoshiotu/2fa#3]
Pulled, built and ran, scaned the code(shows up in the google autheticator app), but when enter those digits, it shows Sorry, Not Authenticated.

Is this repo still working and maintened?

**Comment by yoshiotu** [comment]
It is working for me. Can you please provide more details?

**Comment by IvRRimum** [comment]
What kind of details?

**Comment by IvRRimum** [comment]
I litearary just pulled, scanned the app and tryed to enter the code from my google authoteticator and it fails.

**Comment by yoshiotu** [comment]
You probably already checked this, but are the realtime clocks set correctly, both on your computer and on the mobile device that is running Google Authenticator? Just asking the obvious in case it was a simple oversight.

I am unable to reproduce your problem so far. I ran it successfully on a Mac Mojave and on Ubuntu 18.04, with Go 1.11.*, and used the latest Google Authenticator running on an iPhone. I did `go get -u .` to get the most recent package dependencies. The generated qr.png file has the following checksum/block-count `sum qr.png`: 60808 4.
